content/news/012.dd-ix-at-ripe89.en.md

```diff
@@ -0,0 +1,17 @@
+---
+published: 2024-11-06
+description: Marcel's Lightning Talk and Memorable Experiences
+title: DD-IX at RIPE89 in Prague
+keywords: [ Event ]
+authors: [ Tassilo ]
+image: 012_dd_ix_at_ripe89.webp
+---
+
+Marcel, Klemens, and Melody from DD-IX visited the RIPE community at this year's autumn RIPE event in Prague
```

Comment:
Marcel, Klemens, andMelody participated at the 89th meeting of the RIPE community. RIPE89 took place in Prague.